package.json

```diff
@@ -35,5 +36,17 @@
   "bugs": {
     "url": "https://github.com/lavamoat/LavaDome/issues"
   },
-  "homepage": "https://github.com/lavamoat/LavaDome#readme"
+  "homepage": "https://github.com/lavamoat/LavaDome#readme",
+  "dependencies": {
+    "@lavamoat/preinstall-always-fail": "^2.0.0"
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "@lavamoat/preinstall-always-fail": false,
+      "@wdio/cli>@wdio/utils>edgedriver": false,
+      "chromedriver": false,
+      "esbuild": false,
+      "geckodriver": false
```

Comment:
are you 100% sure the drivers don't need their install scripts? they might need them on a new machine.

Reply:
you are probably right, but if i set these to `true`, what's the benefit I get from allow-scripts eventually?